Saml/Saml.cs

```diff
@@ -14,14 +14,15 @@ version 1.2
 using System.IO.Compression;
```

Comment:
The file originally had tabs, can you convert the whole file to spaces (can do it from one of the VS menus, or a trick is to delete and retype the last } on the page.